crud/upsert.lua

```diff
@@ -18,7 +18,7 @@ local function call_upsert_on_storage(space_name, tuple, operations)
 
     local space = box.space[space_name]
     if space == nil then
-        return nil, UpsertError:new("Space %q doesn't exists", space_name)
+        return nil, UpsertError:new("Space %q doesn't exist", space_name)
```

Comment:
I would like this changes to be performed in a separated patch.

Reply:
Sure, let's push https://github.com/tarantool/crud/pull/45 firstly